components/inputs/input-number.js

```diff
@@ -0,0 +1,83 @@
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { getUniqueId } from '../../helpers/uniqueId.js';
+import { ifDefined } from 'lit-html/directives/if-defined.js';
+import { inputLabelStyles } from './input-label-styles.js';
+import { inputStyles } from './input-styles.js';
+import { offscreenStyles } from '../offscreen/offscreen.js';
+
+class InputNumber extends LitElement {
+
+	static get properties() {
+		return {
+			autocomplete: { type: String },
+			autofocus: { type: Boolean },
+			disabled: { type: Boolean },
+			label: { type: String },
+			labelHidden: { type: Boolean, attribute: 'label-hidden' },
+			max: { type: Number },
+			min: { type: Number },
+			name: { type: String },
```

Comment:
Let's leave `name` off for now since I'm not sure what setting it on the inner input will even do (it's for form submission).

Reply:
Actually... @AllanKerr what do you think here? With `d2l-form-native` I guess it'll do something?

Reply:
I added `name` because the `autocomplete` attribute requires it to map which fields can be autofilled. Not sure if a numeric input would be used for auto-filling things though.

Reply:
`name` is important for form submission. Without it `validate` will still work but the `formValue` won't get included in the submission.

Reply:
Cool OK so sounds like it's worth keeping it around.